### PR TITLE
Site Indicator: update link to disconnect a Jetpack site

### LINKED ISSUE #22360: Site Indicator Action: Disconnect sends folks to the old disconnect flow
When Jetpack sites lose contact, they are prompted to disconnect like this:

image

### What it currently does:
It just opens up a modal:
image

### What it should do:
It should send folks to the disconnect flow initiated in Settings > General > Manage connection:
image

### PR DESCRIPTION
**Pull request by eliorivero**
Fixes #22360

This PR modifies the link to disconnect an unreachable site:

- updates its wording
- points it to the site disconnection flow

captura de pantalla 2018-02-12 a la s 20 01 47

### CODE REVIEW

**Comment by MichaelArestad** [comment]
I think "Remove site" might be better?

**Comment by eliorivero** [comment]
Updated `Delete` to `Remove`

File: client/my-sites/site-indicator/site-indicator.jsx [hunk 1]
```diff
@@ -248,11 +253,15 @@ class SiteIndicator extends Component {
 			accessFailedMessage = (
 				<span>
 					{ translate( 'This site cannot be accessed.' ) }
-					<DisconnectJetpackButton
```
Comment (MichaelArestad) [inline client/my-sites/site-indicator/site-indicator.jsx]: Is this component only used here? If so, should we remove it altogether?

Comment (eliorivero) [reply client/my-sites/site-indicator/site-indicator.jsx]: Not really. It's also used in https://github.com/Automattic/wp-calypso/blob/5f15f3365213253724ca0d8798c6cb8160f00bc9/client/my-sites/plugins/plugin-site-disabled-manage/index.jsx so let's leave it for now.

**Comment by ockham** [comment]
Note that we were deliberately not pointing that button to the Disconnect Survey, see #19491:

> Personally, the one instance where I’m not entirely sure it makes sense to have the survey is the broken Jetpack indicator in the site selector scenario — what answer other than _It was broken, and I was nudged to disconnect_ do we expect?

**Pull request merged**